lib/widgets-page.php

```diff
@@ -76,6 +76,7 @@ function gutenberg_widgets_init( $hook ) {
 		return;
 	}
 
+	gutenberg_enqueue_wp_old_editor_compatibility_snippet();
```

Comment:
Do you think this should be specific to these screens or always applied (for safety). If the latter, we have a dedicate place for this kind of inline scripts in Gutenberg, the `gutenberg_override_script` function

Reply:
Putting it in `gutenberg_override_script` makes sense to me. The post block editor might benefit from this compatibility shim too. At the very least it means there's more consistency.